src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    public String color;
+    public int number;
+
+    @Override
```

Comment:
The 'toString' method is implemented correctly, but make sure to only override it if you need a custom String representation for the class.